app/src/main/java/com/framgia/forder/screen/shopinfo/ShopinfoFragment.java

```diff
@@ -34,9 +38,14 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
             @Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Navigator navigator = new Navigator(getParentFragment().getParentFragment());
+        ManagerShopInfoAdapter adapter = new ManagerShopInfoAdapter(getActivity());
         ShopManagement shopManagement = (ShopManagement) getArguments().get(EXTRA_SHOP);
-        mViewModel = new ShopinfoViewModel(navigator, shopManagement);
-        ShopinfoContract.Presenter presenter = new ShopinfoPresenter(mViewModel);
+        mViewModel = new ShopinfoViewModel(navigator, shopManagement, adapter);
+
+        RealmApi realmApi = new RealmApi();
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to realmApi in com.framgia.forder.screen.shopinfo.ShopinfoFragment.onCreateView(LayoutInflater, ViewGroup, Bundle)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE